examples/components/ex-colorpicker.html

```diff
@@ -0,0 +1,15 @@
+<ex-colorpicker width="100px">
```

Comment:
This is a copy-paste of the inline colorpicker example. Some thinking should happen around making this more consistent (see https://github.com/Tram-One/tram-lite/issues/42)